src/foam/nanos/crunch/ui/MinMaxCapabilityWizardlet.js

```diff
@@ -121,15 +121,6 @@ foam.CLASS({
   ],
 
   methods: [
-    function createView(data) {
```

Comment:
@KernelDeimos @tharmaman 
Just an fyi with this. Tested

Reply:
Thanks! This was outdated